masterlist.yaml

```
@@ -18363,7 +18363,7 @@ plugins:
     after: [ 'Circlets and Masks with all Robes and Hoods.esp' ]
     msg:
       - <<: *patch3rdParty_KPH_WACCF
-        condition: 'active("Weapons Armor Clothing & Clutter Fixes.esp") and not active("imp_helm_legend_WACCF_Patch.esp") and not active("imp_helm_legend_WACCF_AMB_Patch.esp")'
+        condition: 'active("Weapons Armor Clothing & Clutter Fixes.esp") and not active("imp_helm_legend_(BMF_)?WACCF_(AMB_)?_Patch\.esp")'
```

Sorry missed that extra `_` in my review.
```suggestion
        condition: 'active("Weapons Armor Clothing & Clutter Fixes.esp") and not active("imp_helm_legend_(BMF_)?WACCF_(AMB_)?Patch\.esp")'
```